ui/ducks/send/send.js

```diff
@@ -313,12 +318,14 @@ async function estimateGasLimitForSend({
       error.message.includes('Transaction execution error.') ||
       error.message.includes(
         'gas required exceeds allowance or always failing transaction',
-      );
+      ) ||
+      (CHAIN_ID_TO_GAS_LIMIT_BUFFER_MAP[chainId] &&
+        error.message.includes('gas required exceeds allowance'));
```

Comment:
Is this redundant? Or are `gas required exceeds allowance` and `gas required exceeds allowance or always failing transaction` separate and different error messages?